subworkflows/nf-core/abundance_differential_filter/main.nf

```diff
@@ -179,11 +189,20 @@ workflow ABUNDANCE_DIFFERENTIAL_FILTER {
         ch_diff_filter_params.fc_input,
         ch_diff_filter_params.stat_input
     )
+    ch_versions = ch_versions.mix(CUSTOM_FILTERDIFFERENTIALTABLE.out.versions.first())
+
+    ch_dream_results = Channel.empty()
+
+    VARIANCEPARTITION_DREAM(ch_contrasts_dream)
+    ch_versions = ch_versions.mix( VARIANCEPARTITION_DREAM.out.versions.first() )
+    ch_dream_results = VARIANCEPARTITION_DREAM.out.results
```

Comment:
What is the reason for new new method to have its own output channel, when all other methods direct their results to the same `ch_results` channel: